docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -215,10 +231,30 @@ Additionally, inside of the scripts of the modules you will find some internal c
 
 The idea is to keep the modules as generic as possible and only use their features (optionally from an application app). This way, it is easy to upgrade certain modules to newer versions without having to adjust the customizations made before.
 
+As a lot of the modules provide an UI, it can be even possible to solve applications out of the box by just running the modules on a device, connect to them via browser and configure them.
```

Comment:
```suggestion
As a lot of the modules provide a UI. In some cases it can even be possible to solve applications out of the box by just running the modules on a device and configuring them via the UIs by using a web browser.
```

Reply:
Will merge a combination...